packages/patternfly-4/react-core/src/components/ChipGroup/ChipGroup.tsx

```diff
@@ -44,7 +48,8 @@ export class ChipGroup extends React.Component<ChipGroupProps, ChipGroupState>{
 
   renderToolbarGroup() {
     const { isOpen } = this.state;
-    return <InnerChipGroup {...this.props} isOpen={isOpen} onToggleCollapse={this.toggleCollapse} />;
+    const { headingLevel = 'h4' } = this.props;
+    return <ChipGroupContext.Provider value={headingLevel}><InnerChipGroup {...this.props} isOpen={isOpen} onToggleCollapse={this.toggleCollapse} /></ChipGroupContext.Provider>;
```

Comment:
We really need to turn our linter back on 😄  I think this would be more readable as multiple lines, like:

```js
return (
  <ChipGroupContext.Provider value={headingLevel}>
    <InnerChipGroup {...this.props} isOpen={isOpen} onToggleCollapse={this.toggleCollapse} />
  </ChipGroupContext.Provider>
);
```